etc/dissectors/n5305a/meson.build

```diff
@@ -0,0 +1,12 @@
+n5305a_srcs = [
+	'n5305a.cc', 'frameDissector.cc', 'transactDissector.c'
+]
+
+shared_library(
```

Comment:
This should probably be a `shared_module` rather than a `shared_library`.

Also to prevent the `lib` prefix for the dissectors you can use the `name_prefix: '',` argument and set it to an empty string to get rid of it.